clients/python/agentic-sandbox-client/gateway-kind/run-test-kind.sh

```diff
@@ -0,0 +1,98 @@
+#!/bin/bash
+# Copyright 2025 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+set -e
+
+export KIND_CLUSTER_NAME="agent-sandbox"
+
+# Get the image tag using a temporary Python script
+REPO_ROOT=$(git rev-parse --show-toplevel)
+
+TEMP_PY_SCRIPT=$(mktemp /tmp/get_tag_XXXXXX.py)
+cat <<EOF > "$TEMP_PY_SCRIPT"
+import sys
+sys.path.append('$REPO_ROOT/dev/tools/shared')
+from utils import get_image_tag
+print(get_image_tag())
+EOF
+
+IMAGE_TAG=$(python3 "$TEMP_PY_SCRIPT")
+
+rm "$TEMP_PY_SCRIPT"
+echo "Using image tag: $IMAGE_TAG"
+
+
+export SANDBOX_ROUTER_IMG="kind.local/sandbox-router:${IMAGE_TAG}"
+export SANDBOX_PYTHON_RUNTIME_IMG="kind.local/python-runtime-sandbox:${IMAGE_TAG}"
+
+# following develop guide to make and deploy agent-sandbox to kind cluster
+cd $REPO_ROOT
+make build
+make deploy-kind EXTENSIONS=true
+make deploy-cloud-provider-kind
+
+cd clients/python/agentic-sandbox-client/gateway-kind
+echo "Applying CRD for template - Sandbox claim will be applied by the sandbox client in python code"
+sed -i "s|IMAGE_PLACEHOLDER|${SANDBOX_PYTHON_RUNTIME_IMG}|g" python-sandbox-template.yaml
+kubectl apply -f python-sandbox-template.yaml
+
+
+cd ../sandbox-router
+echo "Applying CRD for router template"
+sed -i "s|IMAGE_PLACEHOLDER|${SANDBOX_ROUTER_IMG}|g" sandbox_router.yaml
```

Comment:
Does the image need to be build and loaded to the kind cluster first before applying?